src/main/java/productDependency/CreateOrderPage.java

```diff
@@ -0,0 +1,1481 @@
+			log.info("click on dependenct product.");
+			String DependentProduct = BasePage.getCellData(xlsxName, sheetName, 7, 0);
+			driver.findElement(By.xpath("//a[not(@href)]/strong[text()='" + DependentProduct + "']")).click();
+		}
+		
+	// @FindBy(how=How.XPATH, using="//*[@id="dependencies-products-change_-3"]/tbody/tr[3]/td[1]/a/strong
+	private WebElement clickDependentProduct1;
+
+		 		/**
+		 		 * Method to click on dependent product.
+		 		 * 
+		 		 * @throws IOException
+		 		 */
+	public void clickDependentProduct1() throws IOException {
+		 	CreateOrderPage sp = new CreateOrderPage(driver);
+		 	log.info("click on dependenct product.");
+		 	String DependentProduct = BasePage.getCellData(xlsxName, sheetName, 7, 2);
+		 	driver.findElement(By.xpath("//a[not(@href)]/strong[text()='" + DependentProduct + "']")).click();
+		 		}
+		 		
+	// @FindBy(how=How.XPATH, using="//*[@id="dependencies-products-change_-3"]/tbody/tr[3]/td[1]/a/strong
+	private WebElement clickDependentProduct2;
+
+		 		/**
+		 		 * Method to click on dependent product.
+		 		 * 
+		 		 * @throws IOException
+		 		 */
+	public void clickDependentProduct2() throws IOException {
+		 	CreateOrderPage sp = new CreateOrderPage(driver);
+		 	log.info("click on dependenct product.");
+		 	String DependentProduct = BasePage.getCellData(xlsxName, sheetName, 7, 2);
+		 	driver.findElement(By.xpath("//a[not(@href)]/strong[text()='" + DependentProduct + "']")).click();
+		 		}
+		 		
+	// @FindBy(how=How.XPATH, using="//*[@id="dependencies-products-change_-3"]/tbody/tr[3]/td[1]/a/strong
+	private WebElement clickDependentProduct3;
+
+	 	 		/**
+	 	 		 * Method to click on dependent product.
+	 	 		 * 
+	 	 		 * @throws IOException
+	 	 		 */
+	 public void clickDependentProduct3() throws IOException {
+		 
+	 	 	CreateOrderPage sp = new CreateOrderPage(driver);
+	 	 	log.info("click on dependenct product.");
+	 	 	String DependentProduct = BasePage.getCellData(xlsxName, sheetName, 7, 1);
+	 	 	driver.findElement(By.xpath("//a[not(@href)]/strong[text()='" + DependentProduct + "']")).click();
+	 	 		}
+		
+	 // @FindBy(how=How.XPATH, using="//*[@id="dependencies-products-change_-3"]/tbody/tr[3]/td[1]/a/strong
+	 private WebElement clickDependentProduct4;
+
+	 	 		/**
+	 	 		 * Method to click on dependent product.
+	 	 		 * 
+	 	 		 * @throws IOException
+	 	 		 */
+	 public void clickDependentProduct4() throws IOException {
+		 
+	 	 	CreateOrderPage sp = new CreateOrderPage(driver);
+	 	 	log.info("click on dependenct product.");
+	 	 	String DependentProduct = BasePage.getCellData(xlsxName, sheetName, 7, 0);
+	 	 	driver.findElement(By.xpath("//a[not(@href)]/strong[text()='" + DependentProduct + "']")).click();
+	 	 		}
+	 	 		
+	 // @FindBy(how=How.XPATH, using="//*[@id="dependencies-products-change_-3"]/tbody/tr[3]/td[1]/a/strong
+	 private WebElement clickDependentProduct5;
+
+	 	 		/**
+	 	 		 * Method to click on dependent product.
+	 	 		 * 
+	 	 		 * @throws IOException
```

Comment:
@mayankchhajer Please remove the unused WebElements.

Reply:
Done